Controller/RoleController.php

```diff
@@ -327,4 +335,21 @@ public function deleteRoleAssignmentAction(Request $request, $roleAssignmentId)
 
         return $this->redirectToRouteAfterFormPost('admin_roleView', ['roleId' => $roleAssignment->role->id]);
     }
+
+    /**
+     * Lets you select a section which will be used as limitation when assigning the role.
+     *
+     * @param Request $request
+     * @param int $roleId Role ID
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function assignRoleBySectionAction(Request $request, $roleId)
```

Comment:
`assignRoleBySectionLimitationAction` then, to stay in sync with other *Action methods.